LOGGING.md

```diff
@@ -115,12 +115,37 @@ And, can be deployed without `--dry-run`:
 helm upgrade -f .values.yaml --install cwm-worker-deployment-minio ./helm
 ```
 
+## Helm ConfigMap
+
+An alternative, more flexible, and rather preferable way to configure the logger
+is via its Helm's ConfigMap. The
+[logger-configmap.yaml](./helm/templates/logger-configmap.yaml) can easily be
+configured/reconfigured even for the settings not already exposed via the
+environment variables.
+
+**NOTE**: Only a small subset of the required configurations has been exposed
+via the environment variables.
+
+The logger's ConfigMap provides direct access to the fluentd configuration. The
+log targets can easily be configured for a more complex setup. For more details,
+please visit their respective pages:
+
+- ElasticSearch: https://docs.fluentd.org/output/elasticsearch
+- S3: https://docs.fluentd.org/output/s3
```

Comment:
the helm chart should not allow all possible fluentd configurations to be set, it should only use specific, supported and documented configurations, all based on the helm values.yaml

Reply:
Noted. Will update accordingly.